src/components/atoms/Timeline/ScaleList.tsx

```diff
@@ -0,0 +1,117 @@
+import { memo } from "react";
+
+import { styled } from "@reearth/theme";
+import { XXSRegular } from "@reearth/theme/fonts";
+
+import { EPOCH_SEC, STRONG_SCALE_WIDTH, NORMAL_SCALE_WIDTH, PADDING_HORIZONTAL } from "./constants";
+
+type Props = {
+  gapHorizontal: number;
+} & ScaleListInnerProps;
+
+export const ScaleList: React.FC<Props> = ({ gapHorizontal, ...rest }) => {
+  return (
+    <ScaleContainer style={{ gap: `0 ${gapHorizontal}px` }}>
+      <ScaleListInner {...rest} />
+    </ScaleContainer>
+  );
+};
+
+const MONTH_LABEL_LIST = [
+  "Jan",
+  "Feb",
+  "Mar",
+  "Apr",
+  "May",
+  "Jun",
+  "Jul",
+  "Aug",
+  "Sep",
+  "Oct",
+  "Nov",
+  "Dec",
+];
+
+type ScaleListInnerProps = {
+  start: Date;
+  scaleCount: number;
+  hoursCount: number;
+  scaleInterval: number;
+  strongScaleHours: number;
+};
+
+const ScaleListInner: React.FC<ScaleListInnerProps> = memo(function ScaleListPresenter({
+  start,
+  scaleCount,
+  hoursCount,
+  scaleInterval,
+  strongScaleHours,
+}) {
+  const strongHours = hoursCount * strongScaleHours;
+  const lastStrongScaleIdx = strongHours * (scaleCount / strongHours);
```

Comment:
Can you explain why you are calling things strongXXXX? It might be another translation hiccup, but not sure!

Reply:
`strong` means emphasized scale 🤔
I'm setting width to scale and normal scale should be `1px` or **strong** scale  should be `2px`.
https://github.com/reearth/reearth-web/blob/d5d30090118b411b25aae0b9bacaf493b28a9678/src/components/atoms/Timeline/ScaleList.tsx#L113-L114

Reply:
I am still a little confused what is what from the naming, which might be my lack of understanding for everything that goes in to a geographic scale. For now its fine, but as the component comes closer to completion, especially when I can use it in the PR preview, there is a chance I might ask for the naming to be changed to be clearer!